autoload/lsp/options.vim

```diff
@@ -35,6 +35,7 @@ export var lspOptions: dict<any> = {
   # Make diagnostics show in a popup instead of echoing
   showDiagInPopup: true,
   # Don't print message when a configured language server is missing.
+  diagLineHL: 'Cursor',
   ignoreMissingServer: false,
```

Comment:
The comment revers to `ignoreMissingServers`, but you put `diagLineHL` in between